tests/dbt/package/models/contacts/contacts.yml

```diff
@@ -2,13 +2,56 @@ version: 2
 
 models:
   - name: contacts
+    config:
+      contract:
+        enforced: true
     columns:
-      - name: _id
+      - name: uuid
+        data_type: string
+        constraints:
+          - type: unique
         tests:
           - not_null
+      - name: savedTimestamp
+        data_type: timestamp
+      - name: _deleted
+        data_type: boolean
+      - name: reported
+        data_type: timestamp with time zone
+      - name: parent_uuid
+        data_type: string
+      - name: name
+        data_type: string
+      - name: contact_type
+        data_type: string
+      - name: phone
+        data_type: string
+      - name: phone2
+        data_type: string
+      - name: active
+        data_type: string
+      - name: notes
+        data_type: string
+      - name: contact_id
+        data_type: string
+      - name: edited
+        data_type: string
 
   - name: persons
+    config:
+      contract:
+        enforced: true
     columns:
-      - name: _id
+      - name: uuid
+        data_type: string
+        constraints:
+          - type: foreign_key
+            expression: "{{ env_var('POSTGRES_SCHEMA') }}.contacts (uuid) ON DELETE CASCADE"
```

Comment:
Do we need something similar in `reports.yml`? The deleted reports are not propagating from `v1.medic` to `dbt.Reports`  The deletions of contacts and persons are working fine. See video:

https://github.com/medic/cht-sync/assets/21312057/4409de75-3c43-4225-b9d4-e3e7a89ff4d7

Reply:
After further testing we have confirmed with @lorerod that deletions for reports also work but she had a weird bug where a report was deleted in the CHT but that was not propagated to postgres. We will try and have reproducible steps for that and create an issue once we have those.

Reply:
Okay, I finally figured it out. The report still exists in the database. If I log in as an admin, I can see it.
What happened was that the report's person was deleted before the report. I wonder if this is expected in the CHT. I will ask in another thread.
Here is the video reproducing:

https://github.com/medic/cht-sync/assets/21312057/9a462d61-6189-45da-aeb7-778ee5ca5f99